drizzlepac/drizCR.py

```diff
@@ -45,7 +45,6 @@ def drizCR(input=None, configObj=None, editpars=False, **inputDict):
         run(configObj)
 
 
-# this is the function that will be called from TEAL
```

Comment:
I am not sure why this comment is removed.  This is a similar situation to createMedian.py Line 55.  I do not this it hurts to leave the comment as the function is still invoked.